readthedocs/rtd_tests/tests/test_sync_versions.py

```diff
@@ -391,7 +388,7 @@ def test_update_inactive_stable_version(self):
 
         version_stable = Version.objects.get(slug=STABLE)
         self.assertFalse(version_stable.active)
-        self.assertEqual(version_stable.identifier, '0.9')
+        self.assertEqual(version_stable.identifier, '1.0.0')
```

Comment:
Shouldn't this test be for `0.9`? Or is this test being repurposed to check the new logic?

Reply:
Yea, that was the original fix.